TestFiles cli argument should be the list of relative paths (#257)
cqfn/save: README.md

## Executing specific tests
It can be useful to execute only a number of tests instead of all tests under a particular `save.toml` config.
To do so, you want to pass a relative path to test file after all configuration options:
```bash
$ save [options] /path/to/tests/Test1
```
or a list of relative paths to test files (separated with spaces)
```bash
$ save [options] /path/to/tests/Test1 /path/to/tests/Test2
```
SAVE will detect the closest `save.toml` file and use configuration from there.

> `Note:` On Windows, for execution we need to use double `\\`.
> Command should look like:
> ```bash
> $ save [options] path\\to\\tests\\Test3
> ```
> 

Reviewer: ```suggestion
`Note:` On Windows, you may need to use double backslash `\\` as path separator
```

Author: Fixed
